# My Jupyter Notebook on IBM Data Science Experience

**Name: ROHIT AKOLE**  
I want to be an Data Scientist. I graduated in MS in Technology Management (IT and BigData).

_I'm interested in Data Science because I like to play with the data and statistics. I get excited when it comes to play with numbers and coding. Since my childhood, I have been doing small experiments on the coding languages like Java, C++, VB .Net, etc. I have created, manipulated data using MS Access, SQL, and so on. Hence, I chose this field of interest._

### Following code will store the answer to, an mathematical equation of values stored in x and y, in z and print it

In [1]:
x = 20
y = 28
z = x * y
print('Answer to the multiplication of x and y is = ', z)

Answer to the multiplication of x and y is =  560


#### The above code multiplied x = 15 and y = 23.  
**(horizontal rule)**

---

#### The answer to 
`z = x * y` **(Code/ Syntax Highlighting)**
#### this code is 345
**(horizontal rule)**

***

##### The above code demonstrates following: 
**(bullets)**
- The code is written in Python.
- It gives an accurate answer.
- we can change values of variables and it will give an accurate answer each time.

**(horizontal rule)**

---

#### I tested multiplication function using the above code using following values:
**(table)**

|x|y|z|
|---|---|---|
|15|23|**345**|
|20|28|**560**|